builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -670,7 +672,9 @@ def writeServerlessFile(config, environment_logical_id){
   sh "sed -i -- 's/\${file(deployment-env.yml):subnetIds}/${config['subnetIds'] }/g' serverless.yml"
   sh "sed -i -- '/#Start:resources/,/#End:resources/d' ./serverless.yml"
   sh "sed -i -- '/#Start:isScheduleEnabled/,/#End:isScheduleEnabled/d' ./serverless.yml"
-
+	if ((config['service'] == 'acl' || config['service'] == 'create-serverless-service' || config['service'] == 'delete-serverless-service') && config['domain'] == 'jazz' && configLoader.JENKINS.DOCKERIZED == "true") {
```

Comment:
Can this list of services come from configLoader?

Reply:
Ok, Should it be under the installer vars as a list or as function to output the list?

Reply:
@devsatishm updated the services list from configLoader as functions.